source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,204 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+database. To retrieve data, use read operations.
+
+Read operations allow you to do the following:
+
+- Retrieve a subset of documents from your collection using a :ref:`find operation <retrieve-find>` 
+- Perform transformations on retrieved documents from your collection using an :ref:`aggregate operation <retrieve-aggregate>`
+- Monitor real-time changes to your database using a :ref:`change stream <retrieve-watch>` 
+
+.. _retrieve-paint-order-collection:
+
+Sample Data for Examples
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following sections feature examples of how the owner of a paint
+store manages their customers' orders. For each order, the owner keeps
+track of the color and quantity, which corresponds to the ``color`` and
+``qty`` fields in their ``paint_order`` collection: 
+
+.. code-block:: json
+
+   { "_id": 1, "color": "purple", "qty": 10 }
+   { "_id": 2, "color": "green", "qty": 8 }
+   { "_id": 3, "color": "purple", "qty": 4 }
+   { "_id": 4, "color": "green", "qty": 11 }
+
+Read Operations
+---------------
+
+.. _retrieve-find:
+
+Find
+~~~~
+
+Use the find operation to retrieve a subset of your existing data in
+MongoDB. You can specify what data to return including which documents
+to retrieve, in what order to retrieve them and how many to retrieve. 
+
+To perform a find operation, call the ``find()`` method on an instance
+of a ``MongoCollection``. This method searches a collection for documents that
+match the query filter you provide. For more information on how to
+specify a query, see our :doc:`Specify a Query
+</fundamentals/crud/query-document>` guide. 
+
+Example
+```````
+
+The owner would like to know which orders contain greater than three, but
+less than 10 cans of paint from their :ref:`paint_order collection <retrieve-paint-order-collection>`.
```

Comment:
Comment: I noticed a mix of numbers being used here and it just stood out: three, 10